Wiki instruments (#7555)
aces/Loris: docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/03_instrument_install.md

SQL table on the database thus making their installation much simpler. Once your 
instrument is finalized, save it and move it to the `project/instruments/` directory 
(this is the same location the Instrument Manager stores installed instruments). 
Once the instrument is in the correct location, you must add it manually to the 
`test_names` table of the database to register it. The following query can be used 
as a template for the registration.
  
 ```
  	INSERT INTO test_names (Test_name,Sub_group) SELECT 'TEST_NAME', ID FROM test_subgroups WHERE Subgroup_name='Instruments';
 ```
> _**Note:** The `Sub_group` entry in the query above is necessary for the 
>instrument registration. See See the [Instrument Grouping](#instrument-grouping) 
>section below for additional details._
  		
By completing these steps the instrument should be installed and ready to be 
assigned to a battery; only instruments assigned to a behavioural test battery 
are available for data entry For more information about test batteries, navigate 
to the [Test Battery](./07_clinical_test_battery.md) section directly or continue 
following the documentation.

> > _**Note:** The `TEST_NAME` value int he SQL statement must match exactly the 

Reviewer: ```suggestion
> _**Note:** The `TEST_NAME` value in the SQL statement must match exactly the 
```

## Installation

The steps for installing a LORIS instrument depend on the instrument format chosen (**PHP** or **LINST**) and the data storage type selected (**SQL** table or **JSON**). This document details different installation options for all types of instruments. 

### OPTION 1: SQL Table Storage
#### LINST Instruments - The Instrument Manager 
**The Instrument Manager** module is available to users with the appropriate permissions. This module allows front-end users to upload LINST instruments into the web browser which will then be automatically installed. This module only functions with LINST instruments which are set to use a dedicated SQL table in the database. 

Follow the steps below in order to use the Instrument Manager for your instrument.

  1. Navigate to the Instrument Manager module under the **Admin** menu.
  2. Navigate to the upload tab within the module
  3. Choose browse and select the file you have saved from the Instrument Builder (the file must have a `.linst` extension)
  4. Click "Install" to submit your instrument.

> After installing your instrument, you can load it in a timepoint to test it. For instructions on populating instruments navigate directly to the [Instruments section](TODO) under Getting Started continue following the documentation.

Reviewer: ```suggestion
After installing your instrument, you can load it in a timepoint to test it.
```

This is just confusing.

Author: fixed it. I had forgotten to point to the correct place

  		```
  5. Source the newly generated SQL patch `project/TEST_NAME.sql` into your SQL database to create the table.
  6. Register the instrument in the `test_names` table of the database by running the following command. Make sure to replace the `'TEST_NAME'` value with the actual test name used in your instrument. 
  
  	```
  	INSERT INTO test_names (Test_name,Sub_group) SELECT 'TEST_NAME', ID FROM test_subgroups WHERE Subgroup_name='Instruments';
  	```
  	> _**Note:** The `Sub_group` entry in the query above is necessary for the instrument registration. See the [Instrument Grouping](#instrument-grouping) section below for additional details._
  		
By completing these steps the instrument should be installed and ready to be assigned to a battery; only instruments assigned to a behavioural test battery are available for data entry For more information about test batteries, navigate to the [Test Battery](./07_clinical_test_battery.md) section directly or continue following the documentation.

> _**Note:** Each PHP tool mentioned above contains documentation at the top of the file in case additional details are required._

### OPTION 2: JSON Data Storage

> **PHP and LINST instruments using JSON data storage** do not require a dedicated SQL table on the database thus making their installation much simpler. Once your instrument is finalized, save it and move it to the `project/instruments/` directory (this is the same location the Instrument Manager stores installed instruments). Once the instrument is in the correct location, you must add it manually to the `test_names` table of the database to register it. The following query can be used as a template for the registration.

Reviewer 2: if this is in beta, add that simple caveat (not sure)

Author: It's the recommended path at this stage so not beta
